caddyhttp/httpserver/replacer.go

```diff
@@ -16,6 +16,9 @@ package httpserver
 
 import (
 	"bytes"
+	"crypto/sha1"
```

Comment:
SHA-1 is broken and should not be used. I understand you want to achieve compatibilty with NGINX, but I don´t feel comfortable with it.

Reply:
We can change it to something else if @mholt doesn't want to keep compatibility with Nginx.

Reply:
If I can be so bold as to speak for mholt ☺️ I don’t think he will be worried about compatibility with nginx 

I think if @elcore can make a suggestion of an alternative you should use that

Reply:
> If I can be so bold as to speak for @mholt ☺️ I don’t think he will be worried about compatibility with nginx

Correct!

Reply:
Changed to `sha256` in the last commit.

Reply:
> I think if @elcore can make a suggestion of an alternative you should use that

Thank you!

Reply:
> Changed to sha256 in the last commit.

:+1: